src/pipeline-stages/src/main/python/UDFTransformer.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for information.
+
+import sys
+if sys.version >= '3':
+    basestring = str
+
+from pyspark.ml.param.shared import *
+from mmlspark._UDFTransformer import _UDFTransformer
+from pyspark import keyword_only
+
+from pyspark.ml.util import JavaMLReadable, JavaMLWritable
+from pyspark.ml.wrapper import JavaTransformer, JavaEstimator, JavaModel
+
+from pyspark.ml.common import inherit_doc
+from pyspark.sql.functions import UserDefinedFunction
+from pyspark.ml.common import inherit_doc
+from mmlspark.Utils import *
+
+@inherit_doc
+class UDFTransformer(ComplexParamsMixin, JavaMLReadable, JavaMLWritable, JavaTransformer):
+    """
+
```

Comment:
Why are there so many empty lines in docstrings in this file?

Reply:
I removed them but that is how the auto generated code looks.